posthog/hogql/database/database.py

```diff
@@ -148,16 +148,8 @@ class Database(BaseModel):
         "events",
         "groups",
         "persons",
-        "person_distinct_ids",
-        "session_replay_events",
-        "cohort_people",
-        "static_cohort_people",
-        "log_entries",
-        "app_metrics",
         "sessions",
-        "heatmaps",
         "query_log",
-        "exchange_rate",
```

Comment:
Concern I would have here is that this has been live for so long people might have some queries based on these tables. I would do a check on all saved views to see if any of these tables live in those queries (and/or just do this in a separate PR so it's isolated

Reply:
So the tables are still queryable, they're just not returned in autocomplete or the database schema on the frontend - queries themselves arent affected by this